_sass/_code.scss

```diff
@@ -0,0 +1,24 @@
+.code {
+  border: solid 1px $code-border-color;
```

Comment:
Properties should be ordered background-color, border, border-radius, font-size, margin-bottom